addon/classes/resolve-async-value.js

```diff
@@ -0,0 +1,13 @@
+import { tracked } from '@glimmer/tracking';
+import { Resource } from 'ember-could-get-used-to-this';
+export default  class AsyncProcessResource extends Resource {
```

Comment:
class name should be `ResolveAsync`. 
remove double blank space before `class`.

Reply:
✔️ nice catch!